### ISSUE TITLE
New release to support Intel Macs?

### ISSUE THREAD

**Issue by andyw8** [JoshCheek/seeing_is_believing#168]
Hi,

The [ffi issue](https://github.com/JoshCheek/seeing_is_believing/issues/161) seems to have been resolved – I'd love if there could be an updated release so this can be used on Intel Macs.

Thanks!

(for anyone looking for a temporary workaround: https://rubygems.org/gems/andyw8-seeing_is_believing)

**Comment by Vucius** [comment]
maybe we can manually changing `seeing_is_believing.gemspec`?
the line 22 `s.add_dependency             "ffi",         "~> 1.15`

**Comment by janko** [comment]
Yes, releasing this fix would make a huge difference for me, I currently cannot install `seeing_is_believing` on my M1 Mac. In the past I installed it manually from the repo, but I would prefer not having to do that on every new Ruby version 😛

**Comment by nilcolor** [comment]
```
$ gem install specific_install
$ gem specific_install https://github.com/JoshCheek/seeing_is_believing.git
```
not the best solution but works. and easier than manual process.
